test/main/deepEqual.js

```diff
@@ -1581,6 +1581,35 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 	}
 );
 
+QUnit.test( "Compare primitive objects", function( assert ) {
+	var SafeNumber = Number,
+	SafeString = String,
+	SafeBoolean = Boolean;
```

Comment:
Oops, makes sense. 

We should fix the code style here, that's relatively new on the code style, but we should declare all the variables with a new var statement.

```js
var SafeNumber = Number;
var SafeString = String;
var SafeBoolean = Boolean;
```